.github/workflows/deploy.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: Deploy Develop
         if: github.ref == 'refs/heads/master'
-        run: bash docker/deploy.sh develop
+        run: bash docker/deploy.sh master
```

Comment:
You can remove this if you want to push to `master`. There would no longer be a need for any "special case".